finance/models.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+from candidate.models import SocialMediaModel, Person
+
+
+@python_2_unicode_compatible
+class Committee(SocialMediaModel):
+    COMMITTEE_TYPES = (
+        ('OF', 'Primarily-formed Official Committee'),
+        ('PF', 'Primariliy-formed Committee'),
+        ('IC', 'Independent Committee')
+    )
+    name = models.CharField(max_length=255)
+    address = models.ForeignKey('locality.Address', null=True)
+    locality = models.ForeignKey('locality.Locality', null=True)
+    committee_id = models.CharField(max_length=16, unique=True)
```

Comment:
Good point. `filer_id` could be slightly confusing, as it doesn't match NetFile's `filerId` field exactly, but I think it's a good choice. Thanks!